Back/src/cli/help.rs

```diff
@@ -75,10 +75,10 @@ pub(crate) fn import_help(db: &UserData) {
     println!("{str}");
 }
 
-pub(crate) fn list_files_help(_: &UserData) {
-    println!("Usage: alias 'lf'");
-    println!("    list_files [path]");
-    println!("List files and folders at a given path");
+pub(crate) fn list_files_as_help(_: &UserData) {
+    println!("Usage: alias 'lfa'");
```

Comment:
Longer commands and abstract new commands seems really hard to remember for a new user/any user
(even `lf`) I think we should try to replicate well known commands instead of only creating our owns

Like `ls` in this case (even had `-l` and `-a` can't do any bad)

Reply:
I see the point, but this debug command (`list_files_as`) is still different from ls as it asks for a user. Wouldn't it be misleading to be called ls ?

Reply:
It would be, fair warning, why not:
ls <= as admin
ls [path] <= as admin
ls [path] [user] <= as `user`
or
ls [path] -u [user] <= as `user`


And it was more of a general reflection on the current state of the cli and less a critique of this specific command

Reply:
Just created an issue for this